solr/core/src/java/org/apache/solr/cloud/ReplicateFromLeader.java

```diff
@@ -155,6 +155,38 @@ public static String getCommitVersion(SolrCore solrCore) {
     }
   }
 
+  public static String determinePollInterval(SolrConfig.UpdateHandlerInfo uinfo) {
+    int hardCommitMaxTime = uinfo.autoCommmitMaxTime;
+    int softCommitMaxTime = uinfo.autoSoftCommmitMaxTime;
+    boolean hardCommitNewSearcher = uinfo.openSearcher;
+    String pollIntervalStr = null;
+    if (hardCommitMaxTime != -1) {
+      // configured hardCommit places a ceiling on the interval at which new segments will be
+      // available
```

Comment:
Code comments need to be re-flowed after "tidy" did its work.
When writing comments that flow to this many lines (kind of a lot) consider /* */ style, which I *think* Tidy reflows for you.